### DISCUSSION TITLE
nz-upload 取消[nzAction]必填属性

### DISCUSSION THREAD

**Discussion by lihuabest** [NG-ZORRO/ng-zorro-antd#5331] [Q&A]
## What problem does this feature solve?
文件上传的时候支持预览，但是不一定就默认上传，有些时候业务会自己组合上传数据

## What does the proposed API look like?
[nzAction]="false"，不要默认调用上传逻辑

[a comment by an automated account is not shown]

**Comment by mballoni** [comment]
Was this issue accepted? Looking forward for this feature support.

**Comment by cipchk** [comment]
@mballoni @lihuabest Pls refer to [update manually demo](https://ng.ant.design/components/upload/en#components-upload-demo-upload-manually)

**Comment by mballoni** [comment]
Hi @cipchk , thank you for your response.

I'm unable to use both the preview feature and manual upload at the same time. It is either one OR another. 

Do you have a demo of that combination as well?
